### PR TITLE
keepassxc: update to 2.6.5.

### PR DESCRIPTION
**Pull request by Piraty**
#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/keepassxc/template [hunk 1]
```diff
@@ -23,9 +23,9 @@ maintainer="Piraty <piraty1@inbox.ru>"
 license="GPL-3.0-or-later, BSD-3-Clause, CC0-1.0, LGPL-2.0-only, LGPL-2.1-only,
  LGPL-3.0-or-later, Nokia-Qt-exception-1.1, MIT, BSL-1.0"
 homepage="https://keepassxc.org/"
-changelog="https://raw.githubusercontent.com/keepassxreboot/keepassxc/blob/develop/CHANGELOG.md"
+changelog="https://raw.githubusercontent.com/keepassxreboot/keepassxc/${version}/CHANGELOG.md"
```
Comment (Chocimier) [inline srcpkgs/keepassxc/template:26]: I prefer changelogs from develop so I can look at changes before updating template, and can just paste.
Working one here is https://raw.githubusercontent.com/keepassxreboot/keepassxc/develop/CHANGELOG.md

Comment (Chocimier) [reply srcpkgs/keepassxc/template:26]: It's stale, nevermind

Comment (Chocimier) [reply srcpkgs/keepassxc/template:26]: But 2.6.5 is unstable at the moment: https://github.com/keepassxreboot/keepassxc/releases/tag/2.6.5

Comment (Piraty) [reply srcpkgs/keepassxc/template:26]: it's not unstable
```
<phoerious> It's not shipped downstream automatically and not picked up by our update check. If you package for downstream, don't use pre-releases if you rely on binaries. Source code is fine.
<phoerious> If everything is fine and all binaries are uploaded, we will eventually promote it to release.
<phoerious> Until that happens, don't rely on binaries not to change.
```

Comment (Chocimier) [reply srcpkgs/keepassxc/template:26]: ok

**Pull request merged**